pkg/models/registration.go

```diff
@@ -97,8 +98,10 @@ func (reg Registration) Validate() (bool, error) {
 		reg.Format != FormatAWSJSON &&
 		reg.Format != FormatCSV &&
 		reg.Format != FormatThingsBoardJSON &&
-		reg.Format != FormatNOOP {
-		return false, fmt.Errorf("Format invalid: %s", reg.Format)
+		reg.Format != FormatNOOP &&
+		reg.Format != FormatSenMLJSON {
+
```

Comment:
No need for this extra space.

Reply:
Ok, I will remove it once it is clear how the communication between device services and export service is done (in relation to the type)

Reply:
Space removed